src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/MessageReader.java

```diff
@@ -88,10 +83,12 @@ private void getUnprocessedMessagesFromPlatform() {
                 .map(message -> message.as(PaymentTransactionStateChangedMessage.class))
                 .filter(message -> messageProcessedManager.isMessageUnprocessed(message))
                 .collect(Collectors.toList());
+
+        LOG.info("{} of {} messages are unprocessed", messages.size(), result.size());
```

Comment:
total is logged in [the previous request](https://github.com/commercetools/commercetools-payment-to-order-processor/pull/12/files/974282e3b96e2e5c84f3f30b9c27f748ec38ed28#diff-43e3e2ea57f79fa2f8eb7a25f93dd4d0R98)

updated the log message [here](https://github.com/commercetools/commercetools-payment-to-order-processor/pull/12/commits/14559bd53b319682a3364070a2bf225d19004f82#diff-43e3e2ea57f79fa2f8eb7a25f93dd4d0R87)